S/Laughter Demon Spawning Cleanup + Logging

san7890 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Hey there,

I took a peek at Slaughter/Laughter demon spawning code, and I threw up. I did the following:

* Fixed the messages to not contain spans themselves, and instead pass the span macro through in the to_chats.

* Adds some game logging to whenever one spawns in a s/laughter demon, very cool indeed.

* Removes single-letter bullshit.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

![image](https://user-images.githubusercontent.com/34697715/194438737-66f9af2a-1051-45bf-9e9a-33b12deaa7a5.png)

I'm a fan. Helps administration too because we didn't log for this already.

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
admin: Whenever a player uses a vial of blood/vial of tickles to bring forth a Slaughter Demon or a Laughter Demon (respectively), it will now log to their individual player log, as well as the global game.log.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

san7890 committed:
S/Laughter Demon Cleanup + Logging

Hey there,

I took a peek at Slaughter/Laughter demon spawning code, and I threw up. I did the following:

* Fixed the messages to not contain spans themselves, and instead pass the span macro through in the to_chats.

* Adds some game logging to whenever one spawns in a s/laughter demon, very cool indeed.

* Removes single-letter bullshit.

Fikou commented on code/modules/antagonists/_common/antag_spawner.dm:
```diff
@@ -239,8 +239,8 @@
 	icon = 'icons/obj/wizard.dmi'
 	icon_state = "vial"
 
-	var/shatter_msg = "<span class='notice'>You shatter the bottle, no turning back now!</span>"
-	var/veil_msg = "<span class='warning'>You sense a dark presence lurking just beyond the veil...</span>"
```
i dont like this, people might want to change their spans, why are you limiting them at that
spans are macros anyway just put macros in the var definition

Fikou reviewed (changes requested):
i forgot to request

san7890 committed:
i hope the guy who adds a new demon sheds a tear over this

oh thank god the code is so advanced i can add span_narsie to the shatter message phew this is so fucking cash so fucking money
